Makefile

```diff
@@ -12,7 +12,7 @@
 # language governing permissions and limitations under the License.
 #
 
-GO = go
+GO ?= $(shell which go)
```

Comment:
Instead of this, change the suggested (commented out) `sudo` cmd below to `sudo -E`.  Make the same change where ROOT_CMD=sudo is used in .github action config too.